src/rag/sql_database.py

```
@@ -0,0 +1,36 @@
+        self.path = Path(path)
+        self.conn = sqlite3.connect(self.path)
+        self.conn.execute(
+            "CREATE TABLE IF NOT EXISTS documents (id INTEGER PRIMARY KEY AUTOINCREMENT, text TEXT, metadata TEXT)"
+        )
+        self.conn.commit()
+
+    def add_documents(
+        self, texts: Iterable[str], metadatas: Iterable[dict] | None = None
+    ) -> None:
+        metadatas = metadatas or [{} for _ in texts]
+        for text, meta in zip(texts, metadatas):
```

Ruff (0.11.9)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)